### ISSUE TITLE
System.ArgumentOutOfRangeException was thrown when I tried to compile a cpp file

### ISSUE THREAD

**Issue by LetMeFly666** [PowerShell/PSReadLine#2655]
## Environment

```none
Windows10 PowerShell in Visual Studio Code
```

## Exception report

```none
哎呀，出现问题。请在报告此 Bug 时添加以下详细信息。
在 GitHub 上报告: https://github.com/lzybkr/PSReadLine/issues/new
-----------------------------------------------------------------------
上 200 个密钥:
 6 4 \ b i n \ g + + . e x e Space - g Space F : \ O t h e r A p p s \ P r o g r a m \ 
G i t \ S t o r e \ S t o r e 2 _ W e b _ V a r i o u s \ v a r i o u s \ 计 算 机 C o 
d e \ 仿 D O S 文 件 管 理 \ F i l e . c p p Space - o Space F : \ O t h e r A p p s \ 
P r o g r a m \ G i t \ S t o r e \ S t o r e 2 _ W e b _ V a r i o u s \ v a r i o u s \ 计 算 机 C o d e \ 仿 D O S 文 件 管 理 \ F i l e . e x e Enter
 . \ F Tab Tab Enter
 c l s Enter
 UpArrow

异常:
System.ArgumentOutOfRangeException: 该值必须大于或等于零，且必须小于控制台缓冲区在该维 
度的大小。
参数名: top
实际值是 -1。
   在 System.Console.SetCursorPosition(Int32 left, Int32 top)
   在 Microsoft.PowerShell.PSConsoleReadLine.ReallyRender(RenderData renderData, String defaultColor)
   在 Microsoft.PowerShell.PSConsoleReadLine.ForceRender()
   在 Microsoft.PowerShell.PSConsoleReadLine.HistoryRecall(Int32 direction)
   在 Microsoft.PowerShell.PSConsoleReadLine.PreviousHistory(Nullable`1 key, Object arg)
   在 Microsoft.PowerShell.PSConsoleReadLine.ProcessOneKey(ConsoleKeyInfo key, Dictionary`2 dispatchTable, Boolean ignoreIfNoAction, Object arg)
```

## Steps to reproduce

```none
With two powershell windows, I just use the command `git add .   & git commit -m "push"   & git checkout master   & git merge dev   & git push  & git checkout dev` in one window, 
and `F:\OtherApps\Program\Dev\MinGW64\bin\g++.exe -g F:\OtherApps\Program\Git\Store\Store2_Web_Various\various\计算机Code\仿DOS文件管理\File.cpp -o F:\OtherApps\Program\Git\Store\Store2_Web_Various\various\计算机Code\仿DOS文件管理\File.exe` in another window.
Then the bug happens.
```

## Expected behavior

```none
The source file should be compiled successfully and the corresponding executable file should be generated.
```

## Actual behavior

```none
The Bug happens
```

**Comment by LetMeFly666** [comment]
Thank you so much, and I will learn more

**Comment by LetMeFly666** [comment]
> This issue was already fixed (see #1306). Please upgrade to the [2.1.0 version of PSReadLine](https://www.powershellgallery.com/packages/PSReadLine/2.1.0) from PowerShell Gallery.
> See the [upgrading section](https://github.com/PowerShell/PSReadLine#upgrading) for instructions. Please let us know if you run into the same issue with the latest version.


**Thank you so much, and I will learn more**
